packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -1101,17 +1097,17 @@ class TableRow<T> extends StatefulWidget {
         activeSearchMatchNotifier = null,
         super(key: key);
 
-  final LinkedScrollControllerGroup linkedScrollControllerGroup;
+  final LinkedScrollControllerGroup? linkedScrollControllerGroup;
 
-  final T node;
-  final List<ColumnData<T>> columns;
-  final ItemCallback<T> onPressed;
-  final List<double> columnWidths;
+  final T? node;
+  final List<ColumnData<T>?> columns;
+  final ItemCallback<T>? onPressed;
+  final List<double>? columnWidths;
```

Comment:
non-nullable

Reply:
just two of them.
node has default null and onPressed is optional, so they stay nullable